src/components/RoomForm/RoomJoin/index.js

```diff
@@ -0,0 +1,2 @@
+import RoomJoin from './RoomJoin';
```

Comment:
All of the `index.js` files you see throughout the project just help with the exporting/importing.
For example, App.js only needs to do 
`import RoomForm from '../components/RoomForm';` 
instead of 
`import RoomForm from '../components/RoomForm/RoomForm';`